querygen/userjourney_fs.go

```diff
@@ -27,6 +27,10 @@ var UserJourneyFieldSet = FieldSet{
 		subObj:    "properties",
 		srcFields: []string{"responseObject.metadata.labels.appstudio.openshift.io/component"},
 	},
+	"vulnerabilities_critical": {subObj: "properties", srcFields: []string{"clair_scan_result.vulnerabilities.critical"}},
```

Comment:
Try keeping the line length in the code below 100 characters or so so GitHub does not end up word-wrapping the code.

Reply:
@ifireball I have updated the code. Ty.